Client/src/Pages/Monitors/CreateMonitor/index.jsx

```diff
@@ -84,11 +84,11 @@ const CreateMonitor = () => {
     event.preventDefault();
     //obj to submit
     let monitor = {
-      ...generalSettings,
       url:
         checks.type === "http"
           ? "https://" + generalSettings.url
           : generalSettings.url,
+      name: generalSettings.name || generalSettings.url,
```

Comment:
Can we replace this with a ternary?  I am really not a fan of falsey evaluation, it makes my skin crawl :P

`generalSettings.name === "" ? generalSettings.url : generalSettings.name`

is a bit more verbose but it is very clear as to what the intent is and there can't be any unexpected evaluations